examples/tutorial-standalone/main.tf

```diff
@@ -0,0 +1,198 @@
+  wrapped_key               = local.wrapped_key_secret_data
+  dlp_location              = local.location
+  template_id_prefix        = "re_identification"
+  template_file             = "${path.module}/templates/reidentification.tmpl"
+  dataflow_service_account  = module.secured_data_warehouse.confidential_dataflow_controller_service_account_email
+}
+
+resource "google_artifact_registry_repository_iam_member" "docker_reader" {
+  provider = google-beta
+
+  project    = module.template_project.project_id
+  location   = local.location
+  repository = "flex-templates"
+  role       = "roles/artifactregistry.reader"
+  member     = "serviceAccount:${module.secured_data_warehouse.dataflow_controller_service_account_email}"
+}
+
+resource "google_artifact_registry_repository_iam_member" "confidential_docker_reader" {
+  provider = google-beta
+
+  project    = module.template_project.project_id
+  location   = local.location
+  repository = "flex-templates"
+  role       = "roles/artifactregistry.reader"
+  member     = "serviceAccount:${module.secured_data_warehouse.confidential_dataflow_controller_service_account_email}"
+}
+
+resource "google_artifact_registry_repository_iam_member" "python_reader" {
+  provider = google-beta
+
+  project    = module.template_project.project_id
+  location   = local.location
+  repository = "python-modules"
+  role       = "roles/artifactregistry.reader"
+  member     = "serviceAccount:${module.secured_data_warehouse.dataflow_controller_service_account_email}"
+}
+
+resource "google_artifact_registry_repository_iam_member" "confidential_python_reader" {
+  provider = google-beta
+
+  project    = module.template_project.project_id
+  location   = local.location
+  repository = "python-modules"
+  role       = "roles/artifactregistry.reader"
+  member     = "serviceAccount:${module.secured_data_warehouse.confidential_dataflow_controller_service_account_email}"
+}
+
+module "regional_deid" {
+  source = "../../modules/dataflow-flex-job"
+
+  project_id              = module.base_projects.data_ingestion_project_id
+  name                    = "dataflow-flex-regional-dlp-deid-job-python-query"
+  container_spec_gcs_path = module.template_project.python_re_identify_template_gs_path
+  job_language            = "PYTHON"
+  region                  = local.location
+  service_account_email   = module.secured_data_warehouse.dataflow_controller_service_account_email
+  subnetwork_self_link    = module.base_projects.data_ingestion_subnets_self_link
+  kms_key_name            = module.secured_data_warehouse.cmek_data_ingestion_crypto_key
+  temp_location           = "gs://${module.secured_data_warehouse.data_ingestion_dataflow_bucket_name}/tmp/"
+  staging_location        = "gs://${module.secured_data_warehouse.data_ingestion_dataflow_bucket_name}/staging/"
+
+  parameters = {
+    query                          = "SELECT * FROM [bigquery-public-data:irs_990.irs_990_ein] LIMIT 10000"
```

Comment:
I think it'll be important to highlight with a comment where this public data set is defined (i.e add a link).
Also, best practice wouldn't be to `SELECT *` every column as it would have performance and cost implications.  We can reduce the "schema" definition, as well.  For instance, maybe we pick all the columns we'll transforma and maybe 1 or 2 columns that we won't really use, but just to show how some columns can be processed without DLP.